sdk/src/main/java/io/honeycomb/opentelemetry/OpenTelemetryConfiguration.java

```diff
@@ -339,38 +394,39 @@ public OpenTelemetry build() {
                     EnvironmentConfiguration.HONEYCOMB_API_KEY));
             }
 
-            SpanExporter exporter = builder.build();
-
-            SdkTracerProviderBuilder tracerProviderBuilder = SdkTracerProvider.builder()
-                .setSampler(sampler);
-            this.additionalSpanProcessors.forEach(tracerProviderBuilder::addSpanProcessor);
-
-            if (this.enableDebug) {
-                tracerProviderBuilder.addSpanProcessor(SimpleSpanProcessor.create(LoggingSpanExporter.create()));
-            }
-            tracerProviderBuilder.addSpanProcessor(BatchSpanProcessor.builder(exporter).build());
+            return builder.build();
+        }
 
-            DistroMetadata.getMetadata().forEach(resourceAttributes::put);
-            if (StringUtils.isNotEmpty(serviceName)) {
-                resourceAttributes.put(EnvironmentConfiguration.SERVICE_NAME_FIELD, serviceName);
+        private SpanExporter createHttpSpanExporter(Logger logger) {
+            OtlpHttpSpanExporterBuilder builder = OtlpHttpSpanExporter.builder();
+            if (tracesEndpoint != null) {
+                builder.setEndpoint(tracesEndpoint);
+            } else {
+                builder.setEndpoint(EnvironmentConfiguration.DEFAULT_HONEYCOMB_ENDPOINT);
```

Comment:
We will need to append the endpoint here with `/v1/traces` when using the HTTP span exporter

Reply:
Hmm, I was sure I'd tested that, but on review it doesn't work 🤷🏻